src/ralph/api/routers/statements.py

```diff
@@ -326,6 +326,15 @@ async def get(
             json.loads(query_params["agent"])
         )
 
+    if not mine:
+        if settings.LRS_RESTRICT_BY_AUTHORITY:
```

Comment:
```suggestion
    if settings.LRS_RESTRICT_BY_AUTHORITY:
```
Could this work too (with less nesting)?

Reply:
Good point ! Changed this.